conf.json

```diff
@@ -0,0 +1,6 @@
+{
+  "src_table": "info",
+  "dest_table": "info",
+  "column_mapping": {"ID_": "id", "DURATION_": "start_time"},
+  "column_transformation": {"DURATION_": "toMilliseconds"}
```

Comment:
In my example it's assumed to be seconds as input. Ideally you would have multiple default functions for the most common time units. For more complex operations there's the ability to create custom functions, which I haven't added yet.